browser-test/src/question_lifecycle.test.ts

```diff
@@ -524,4 +528,218 @@ describe('normal question lifecycle', () => {
       'my_test_question',
     )
   })
+
```

Comment:
*suggestion (blocking)*: imo we should move these into a new primary_applicant_info_test file. a) because we'll have a bunch of other PAI related tests with Rocky's work and b) this file is getting too big and c) even though some of these tests are about ensuring that creating a draft, publishing, creating another draft, etc does the right thing, that's not the core goal of these tests.